utils/coreutils/cmdutils.go

```diff
@@ -216,6 +216,11 @@ func ExtractStringOptionFromArgs(args []string, optionName string) (cleanArgs []
 	return
 }
 
+func ExtractUseNativeFromArgs(args []string) (cleanArgs []string, useNpmrc bool, err error) {
+	return extractBoolOptionFromArgs(args, "client-native")
+
+}
+
```

Comment:
```suggestion
func ExtractUseNativeFromArgs(args []string) (cleanArgs []string, useNpmrc bool, err error) {
	return extractBoolOptionFromArgs(args, "client-native")
}

```
remove redundant new line after return statement